### DISCUSSION TITLE
Just for fun: Pocket Icon Creator

### DISCUSSION THREAD

**Discussion by benallfree** [pocketbase/pocketbase#5758] [Show and tell]
Here's a GPT I made to help create app icons and favicons that stay on theme with the pocketbase/golang gopher.

https://chatgpt.com/g/g-Kvm1VgMmr-pocket-icon-creator

Examples:

### "I'm making a crypto trading app"

![image](https://github.com/user-attachments/assets/ba3e4a7f-b7b8-4723-9905-1a865b691ef6)

### "I'm making a social network for homeless silicon valley software devs"

![image](https://github.com/user-attachments/assets/c2296286-d485-49ed-be7c-a24afc7f2a4e)

**Comment by GSGSDgggdzez** [comment]
very cool
